generate_reports.sh

```diff
@@ -40,7 +36,7 @@ if [ -f $family_vcf ]; then
 	cp ../${family_vcf} .
 	# for exome reports, need variant tables from the four variant callers
 	cp ../*table . 
-	vcf2cre_job="$(qsub "${script}" -v original_vcf="${family_vcf}",project=${family})"
+	vcf2cre_job="$(qsub "${script}" -v original_vcf="${family_vcf}",project=${family},ped=${ped})"
```

Comment:
wgs reports may not need ped file, so could we make it conditional on `ped` or `type`?

Reply:
fixed in 3dea37ece21380027e07e863237fa01fd11e74ca
To generate a de novo report (without regenerating annotations) if pedigree file is available: 
`qsub ~/cre/cre.sh -v family=<family>,type=denovo,ped=<pedfile>`

To regenerate annotations and generate de novo report:
`generate_reports.sh <family> denovo <ped>`